addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -281,65 +281,52 @@
-                                        <label>Name *</label>
-                                        <input class="form-control" type="text" t-attf-name="#{counter}-name" required="This field is required"
-                                            t-att-value="default_first_attendee.get('name', '') if counter == 1 else ''"/>
-                                    </div>
-                                    <div class="col-lg my-2">
-                                        <label>Email *</label>
-                                        <input class="form-control" type="email" t-attf-name="#{counter}-email" required="This field is required"
-                                            t-att-value="default_first_attendee.get('email', '') if counter == 1 else ''"/>
-                                    </div>
-                                    <div class="col-lg my-2">
-                                        <label>Phone <small>(Optional)</small></label>
-                                        <input class="form-control" type="tel" t-attf-name="#{counter}-phone"
-                                            t-att-value="default_first_attendee.get('phone', '') if counter == 1 else ''"/>
-                                    </div>
-                                    <input class="d-none" type="text" t-attf-name="#{counter}-event_ticket_id" t-attf-value="#{ticket['id']}"/>
-                                </div>
-                                <t t-if="event.specific_question_ids">
-                                    <div t-foreach="event.specific_question_ids" t-as="question" class="row" t-att-name="question.title">
-                                        <div class="col-lg-6 mt-2">
-                                            <t t-call="website_event.registration_event_question">
-                                                <t t-set="registration_index" t-value="counter"/>
+                    <t t-if="availability_check">
+                        <t t-set="counter" t-value="0"/>
+                        <t t-set="input_type" t-value="{'name': 'text', 'email': 'email', 'phone': 'tel', 'company_name': 'text'}"/>
+                        <t t-set="type_labels" t-value="dict(env['event.question']._fields['question_type'].selection)"/>
+                        <t t-foreach="tickets" t-as="ticket">
+                            <t t-foreach="range(1, ticket['quantity'] + 1)" t-as="att_counter" name="attendee_loop">
+                                <t t-set="counter" t-value="counter + 1"/>
+                                <div class="modal-body">
+                                    <h5 t-attf-class="mt-1 pb-2 #{'border-bottom' if event.question_ids else ''}">Ticket #<span t-out="counter"/> <small class="text-muted">- <span t-out="ticket['name']"/></small></h5>
+                                    <t t-if="event.specific_question_ids">
+                                        <div class="row">
```

Comment:
Note that you can put the t-if into the div to same some spacing and indentation levels.